commit 59c4209152db8eb06ce36664b21a5a21b6adc418
Author: Roman Pen <roman.penyaev@profitbricks.com>
Date:   Sun Jan 8 21:00:35 2017 -0500

    ext4: do not polute the extents cache while shifting extents
    
    commit 03e916fa8b5577d85471452a3d0c5738aa658dae upstream.
    
    Inside ext4_ext_shift_extents() function ext4_find_extent() is called
    without EXT4_EX_NOCACHE flag, which should prevent cache population.
    
    This leads to oudated offsets in the extents tree and wrong blocks
    afterwards.
    
    Patch fixes the problem providing EXT4_EX_NOCACHE flag for each
    ext4_find_extents() call inside ext4_ext_shift_extents function.
    
    Fixes: 331573febb6a2
    Signed-off-by: Roman Pen <roman.penyaev@profitbricks.com>
    Signed-off-by: Theodore Ts'o <tytso@mit.edu>
    Cc: Namjae Jeon <namjae.jeon@samsung.com>
    Cc: Andreas Dilger <adilger.kernel@dilger.ca>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1277_59c4209152db8eb06ce36664b21a5a21b6adc418.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1277_59c4209152db8eb06ce36664b21a5a21b6adc418.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
